### PR TITLE
Shorten types and add missing types

### LINKED ISSUE #272: Second-level types are not included in context/shortened
http://schema.org/Place, http://schema.org/PostalAddress, http://schema.org/OpeningHoursSpecification

I think we should shorten them like the first-level types.

### PR DESCRIPTION
**Pull request by acka47**
Will resolve #272, #273, #274.

### CODE REVIEW

**Comment by acka47** [comment]
@fsteeg Tests resulted in strange error:

```
errorTest transformation.TestMorphss.EnrichedMorph failed: org.culturegraph.mf.exceptions.ValidationException: No entity found: label
```

I haven't changed anything with regard to `label` and, thus, don't understand this.

File: conf/context.jsonld [hunk 1]
```diff
@@ -1,101 +1,106 @@
...
+    "Publisher": "http://vivoweb.org/ontology/core#Publisher",
+    "Organization": "http://schema.org/Organization",
+    "Collection": "http://purl.org/ontology/bibo/Collection",
+    "Concept": "http://www.w3.org/2004/02/skos/core#Concept",
+    "OpeningHoursSpecification": "http://schema.org/OpeningHoursSpecification",
+    "PostalAddress": "http://schema.org/PostalAddress",
+    "Place": "http://schema.org/Place",
+    "GeoCoordinates": "http://schema.org/GeoCoordinates"
+  }
+}
```
Comment (fsteeg) [inline conf/context.jsonld]: You changed the formatting here, which makes it impossible to see the actual difference.

Comment (acka47) [inline conf/context.jsonld]: Oups, did that again. I justz added lines 100 to 104...

**Comment by fsteeg** [comment]
Hm, you did add the type to the tests... weird.

**Comment by fsteeg** [comment]
The test file was missing `type` literals in two positions (see 0faa9447fa0566b2a0b04e306023c650a1b29bfd), so where it was looking for a `label`, it did not find one (got the type there instead), thus the error about a `label`.

**Pull request merged**